examples/demo_app/package.json

```diff
@@ -7,14 +7,14 @@
     "turbolinks": "^5.2.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.4.0",
-    "@babel/preset-env": "^7.4.2",
-    "babel-loader": "^8.0.5",
-    "glob": "^7.1.3",
-    "webpack": "^4.29.6",
+    "@babel/core": "^7.5.5",
+    "@babel/preset-env": "^7.5.5",
+    "babel-loader": "^8.0.6",
+    "glob": "^7.1.4",
+    "webpack": "^4.39.3",
     "webpack-assets-manifest": "^3.1.1",
-    "webpack-cli": "^3.3.0",
-    "webpack-dev-server": "^3.2.1"
+    "webpack-cli": "^3.3.7",
+    "webpack-dev-server": "^3.8.0"
```

Comment:
These upgrades aren't necessary for rails 6, but it doesn't hurt to upgrade now.